tensorflow_addons/activations/README.md

```diff
@@ -4,25 +4,26 @@
 | Submodule | Maintainers               | Contact Info                             |
 |:----------|:--------------------------|:-----------------------------------------|
 | gelu      | @AakashKumarNain @WindQAQ | aakashnain@outlook.com windqaq@gmail.com |
-| hardshrink| @WindQAQ                  | windqaq@gmail.com
+| hardshrink| @WindQAQ                  | windqaq@gmail.com                        |
+| lisht     | @WindQAQ                  | windqaq@gmail.com                        |
 | sparsemax | @AndreasMadsen            | amwwebdk+github@gmail.com                |
-| tanhshrink | @fsx950223            | fsx950223@gmail.com                |
+| tanhshrink| @fsx950223                | fsx950223@gmail.com                      |
 
 ## Contents
 | Submodule | Activation | Reference                        |
 |:----------|:-----------|:---------------------------------|
 | gelu      | gelu       | https://arxiv.org/abs/1606.08415 |
 | hardshrink| hardshrink |                                  |
-| sparsemax | Sparsemax  | https://arxiv.org/abs/1602.02068 |
-| tanhshrink | Tanhshrink  |  |
+| lisht     | lisht      | https://arxiv.org/abs/1901.05894 | 
+| sparsemax | sparsemax  | https://arxiv.org/abs/1602.02068 |
+| tanhshrink| tanhshrink |                                  |
 
 
 ## Contribution Guidelines
 #### Standard API
 In order to conform with the current API standard, all activations
 must:
  * Be a `tf.function`.
- * Have the signature `fn(input, axis=-1, name=None)`.
```

Comment:
I think it would be better to remove it (at least the style is consistent in the same submodule). But not sure if other operations in `image` and `text` etc should keep it.

Reply:
I prefer to keep `name` in other modules :-)

Reply:
Sure, I will create another PR to clean up activation functions (name arg, duplicated tests etc.)